### PR TITLE
added build status of pull request

### PR DESCRIPTION
**Pull request by darkship**
Get build status from last commit of pull request

### CODE REVIEW

File: src/ts/core/interfaces.ts [hunk 1]
```diff
@@ -177,3 +177,12 @@ export interface PullRequestActivity {
   // added?
   // removed?
 }
+
+export interface BuildStatus {
+  state: string,
```
Comment (aalpern) [inline src/ts/core/interfaces.ts]: Looks like there's a spurious comma here

File: src/ts/core/interfaces.ts [hunk 2]
```diff
@@ -177,3 +177,12 @@ export interface PullRequestActivity {
...
+  key: string
+  name: string
+  url: string
+  description: string,
```
Comment (aalpern) [inline src/ts/core/interfaces.ts]: and here

File: src/ts/core/interfaces.ts (hunk 1 above)
Comment (darkship) [reply src/ts/core/interfaces.ts]: fixed

File: src/ts/core/interfaces.ts (hunk 2 above)
Comment (darkship) [reply src/ts/core/interfaces.ts]: fixed

**Review by aalpern — APPROVED** [review]

**Comment by aalpern** [comment]
I'd almost forgotten about this project :)

**Comment by darkship** [comment]
Thanks for your work :)

**Pull request merged**